### PR TITLE
conda recipes for pypif and citrination-client

### PR DESCRIPTION
**Pull request by eddotman**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by maxhutch** [comment]
Looks good to me.

**Comment by sparadiso** [comment]
LGTM

**Comment by jsomara** [comment]
+1 

>  GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

I am willing to be listed there.

File: recipes/pypif/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pypif
+  requires:
```
Comment (chrisburr) [inline recipes/pypif/meta.yaml]: If these are required to import `pypif` should they be runtime dependencies instead?

File: recipes/citrination-client/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - citrination_client
+  requires:
```
Comment (chrisburr) [inline recipes/citrination-client/meta.yaml]: If these are required to import `citrination_client` should they be runtime dependencies instead?

File: recipes/citrination-client/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  sha256: e20fbbac5339153707a8976e68ce7d64f9768aace9278d7e189791abd4d7d2e4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (chrisburr) [inline recipes/citrination-client/meta.yaml]: Do you actually need a C compiler, it looks like this is pure Python? If you do need one this probably can't be a noarch package.
(same applies for `pypif`)

Comment (eddotman) [reply recipes/citrination-client/meta.yaml]: Thanks -- good call!

File: recipes/citrination-client/meta.yaml (hunk 2 above)
Comment (eddotman) [reply recipes/citrination-client/meta.yaml]: Agreed

File: recipes/pypif/meta.yaml (hunk 1 above)
Comment (eddotman) [reply recipes/pypif/meta.yaml]: Agreed

**Pull request merged**